external-providers/generic-external-provider/go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/konveyor/generic-external-provider
 
-go 1.19
+go 1.20
 
 require (
 	github.com/bombsimon/logrusr/v3 v3.1.0
 	github.com/getkin/kin-openapi v0.118.0
 	github.com/go-logr/logr v1.2.4
-	github.com/konveyor/analyzer-lsp v0.3.0-beta.2
+	github.com/konveyor/analyzer-lsp v0.3.2-0.20240424154435-a6985630be6b
```

Comment:
I might be wrong, but isn't this analyzer dependency too frech for release-0.3?

Reply:
it is 0.3.2 which is latest on 0.3